docs/user-guide/workflow-setup.md

```diff
@@ -156,6 +156,10 @@ Sets the [_Browser Profile_](../browser-profiles) to be used for this crawl.
 
 Will prevent any content from the domains listed in [Steven Black's Unified Hosts file](https://github.com/StevenBlack/hosts) (ads & malware) from being captured by the crawler.
 
+### Custom User Agent
+
+Sets the browser's user agent in outgoing requests to the specified value. If not set, the crawler will use the browser's default user agent.
```

Comment:
```suggestion
Sets the browser's user agent in outgoing requests to the specified value. If left blank, the crawler will use the browser's default user agent.
```

Should we link to something like https://www.useragents.me/ to give people a resource to go to for obtaining these?  Should we do this in the app?

Reply:
I think if someone's at the point that they want to specify a custom user agent, we probably don't have to tell them how they work? Just my 2c though.

Reply:
You're not wrong.... But maybe the real answer is we add some common ones that might help people and let them add a custom string if need be?  Most of what I use these for is getting around sites that say "this doesn't work in Firefox" when in reality it's completely fine :P 

Personally I know what a user agent is, I don't know the syntax for specifying them, I only interact with them through browser extensions, I guess I can Google it and find one but not the best experience?

<img width="276" alt="Screenshot 2023-12-20 at 3 58 40 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/5672810/eb151ad2-eba6-40c1-93dd-d5990d312f69">

In any case, lets come back to this later.  Good feature and good first implementation! 🚀